### DISCUSSION TITLE
Is it possible to transfer the authorship of a kata?

### DISCUSSION THREAD

**Discussion by david-posadas** [codewars/codewars.com#2934] [General]
Currently I have several close profiles that designed quite interesting kata.

Soon they will become inactive since they have finished their training period and therefore these katas have been suspended until certain possible problems are answered.

I wouldn't want that work to be lost, so I'd be willing to maintain it so it would continue to be available to the community.

I can't find any option, or I haven't been able to find it, to be able to solve this issue.

**Comment by Madjosz** [comment]
You cannot transfer ownership of a kata. There's only the option that the authors allow contributors in the edit panel:
![Allow Contributors](https://user-images.githubusercontent.com/28844868/217298865-5040e1fd-a0c6-42e9-8943-21f17ca91978.jpg)

On the other hand if issues are raised on the kata anybody (including you) who solved the kata can provide a fix via the "Fork" functionality which then is visible in the translation tab and can be reviewed by other solvers/menders/mods.

**Comment by Blind4Basics** [comment]
I'm not really sure I understand the concept of "close profile" => friends?

About their kata: if they leave while their kata are in draft state, those will be lost. If they are published, they can be found through the kata search page, yes.

**Reply by david-posadas** [reply]
They are profiles of my students who have already left the nest.

Some are already accepted.

**Reply by Blind4Basics** [reply]
Ok. If the kata are published, they just have to "allow contributors", as Madjosz stated, to ease the process. But even if they don't, there are still ways to maintain them (it's just that less people can do it).

**Reply by david-posadas** [reply]
I followed the process that was indicated to me, but it seems that something must not have worked correctly or someone authorized did not accept the changes. 
I'm not quite sure how to proceed.

**Reply by Blind4Basics** [reply]
could you be more specific?
* what kata?
* what translation?
* what does seem to have happened?

**Reply by david-posadas** [reply]
Sorry, I wasn't as clear as I needed to be.

The kata: https://www.codewars.com/kata/635a4f8ce664ce00648fd013

The translation https://www.codewars.com/kata/635a4f8ce664ce00648fd013/forks/python

The problem was that the kata had a little problem with test, and the author goes away from the platform. I want to preserve the work done and do the mantenaince of some katas.

Now no one acepts my changes, and the kata continues in beta mode.

**Reply by Blind4Basics** [reply]
You need to post a `suggestion` in the kata discourses page, with the link to the translation (possibly summarizing why/what you changed). Otherwise nobody will see it and chances it gets reviewed are close to zero.

"additional points" for posting on CW discord as well (in `#fixing` or `#reviewing`, depening on the purpose)
